scripts/setup-dev.sh

```
@@ -0,0 +1,149 @@
+    
+    echo "✅ Dependencies installed"
+}
+
+# Function to setup database
+setup_database() {
+    echo "🗄️  Setting up database..."
+    
+    # Wait for PostgreSQL to be ready
+    echo "⏳ Waiting for PostgreSQL to be ready..."
+    timeout=60
+    while ! docker exec $(docker ps -q -f "name=postgres") pg_isready -U ats_user -d ats_db &> /dev/null; do
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)